lib/generator/command_line.rb

```diff
@@ -11,13 +11,13 @@ def initialize(paths)
     def parse(args)
       parser = GeneratorOptparser.new(args, @paths)
       @options = parser.options
-      generator if parser.options_valid?
+      return generators if parser.options_valid?
```

Comment:
I realize why the return is here, but with the plural form, this is clear even without the return explicitly there.